### DISCUSSION TITLE
Allow multi word room names, when put in quotes?

### DISCUSSION THREAD

**Discussion by 456dev** [Zrips/CMI#3924] [Feature Requests]
**Description of feature/improvement request:**

to allow players or server admins (permission?) to make multi word chat room names when placed in quotes
---
### Detailed explanation of what it should do, when and in what situation: ###
when creating a chatroom, create it multi word , separated by quotes (", or ')
also joining and whatever

### Basic explanation as to why this feature is needed and/or useful to have: ###
words, or separated "channels"

this is really not needed at all
just an alternative to choosing the last argument in the command

![image](https://user-images.githubusercontent.com/42748092/89771127-bcf26180-daf7-11ea-9181-f8713c852892.png)
![image](https://user-images.githubusercontent.com/42748092/89771142-c54a9c80-daf7-11ea-81fc-7d346b62843c.png)

**Comment by CloudeLecaw** [comment]
If this will be added then pls also add ir think about limitation, else we will find rooms like "thus is a room at you never want to enter"... 😅
